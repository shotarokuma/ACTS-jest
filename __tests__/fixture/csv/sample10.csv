Column1;Column2;Column3;Column4;Column5;Column6;Column7;Column8;Column9;Column10;Column11;Column12;Column13;Column14;Column15;Column16;Column17;Column18;Column19;Column20;Column21;Column22;Column23;Column24;Column25;Column26;Column27;Column28;Column29;Column30
13;28;76;;;;;22;67;6;;;78;9;37;;;19;;54;86;26;;;26;;;;58;
;;78;35;0;;;;16;86;68;;83;;24;62;3;;84;16;28;34;0;;;50;61;72;;83
26;7;10;34;95;;64;;4;;;;22;4;;;;20;;;;16;12;44;29;;;35;67;
4;25;61;;22;;;;;55;;22;;;17;;;;;40;35;;;;;;;84;9;
81;;;;43;51;84;;27;32;;;59;29;4;;;98;;;99;53;;;;;;;8;9
;33;60;11;32;86;29;57;37;5;;;56;;;49;60;11;6;67;;;47;;65;;77;;3;
94;60;24;51;52;;;88;;18;;;55;75;19;;;70;;;93;;64;;;17;77;52;87;
40;;;;;;76;;96;53;;;;;;90;9;;50;20;;75;;;78;23;67;;79;
;;70;;;;4;64;54;;;12;;57;45;44;84;;16;;;12;86;;;29;43;;57;
;;;79;;75;4;4;;49;78;2;;;;29;61;;;74;;;73;;;53;9;;;
35;94;68;29;;8;87;;44;;;89;54;;7;;;47;81;88;71;;;38;26;82;;13;35;48
;58;;;20;;;1;;47;5;86;28;;;90;;;;;;97;31;;8;;54;;17;67
2;40;;57;;59;;;;27;;42;;53;62;;;55;85;28;;27;44;;96;;22;58;84;80
20;74;19;;47;;20;51;12;41;;98;99;;;;;;50;55;;;;;;88;86;;60;94
;44;;8;23;;;;45;;;;;;;16;47;86;16;;53;55;93;76;95;35;35;;;57
97;;5;25;;;;86;90;;;29;29;;;20;31;9;20;88;;24;81;;;33;95;7;;
;;6;44;;66;;71;27;;82;;;;;;;42;73;;72;;28;;38;32;;81;89;
1;;86;5;48;50;;;;51;25;;19;45;;;;93;68;75;;39;41;77;20;85;81;33;;
;56;36;50;;45;;55;;;56;;70;43;;;;;;22;;86;;;;46;41;;38;29
;64;83;46;;4;;;;98;;;;71;95;42;33;93;70;;49;39;;99;11;14;;;84;
25;;;18;;11;;85;;;96;31;;18;83;;;;;;2;49;;14;;64;99;;29;63
27;31;;80;;;;;;59;82;27;41;;;;;;;12;98;77;61;;;18;;20;;
66;69;62;63;;;;;42;;;;97;44;;60;;96;;;;;;37;60;67;78;;7;
;;74;;73;;;;;59;73;;;;99;;;;68;15;;;;;;21;;;73;
0;;25;10;74;;;52;;31;;0;59;62;;;79;;;;;9;;85;;26;;;97;
;65;88;81;16;;77;1;57;83;;81;;45;;70;;;;82;39;82;;;;;22;;16;
;65;;;38;71;;20;88;;;50;4;83;18;;41;;;23;24;;87;;18;;22;52;;
;79;;;;;;;;82;12;;16;11;93;10;;;92;89;;;44;;;41;53;;;
;;82;;56;72;;;19;;33;;;;;78;;;93;;;;;;;79;;;15;
;54;4;14;;45;94;22;;;;45;;;;1;51;;59;51;67;;;72;8;24;62;;;
