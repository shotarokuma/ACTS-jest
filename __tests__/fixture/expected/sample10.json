[
{"Column1":"13","Column2":"28","Column3":"76","Column4":"","Column5":"","Column6":"","Column7":"","Column8":"22","Column9":"67","Column10":"6","Column11":"","Column12":"","Column13":"78","Column14":"9","Column15":"37","Column16":"","Column17":"","Column18":"19","Column19":"","Column20":"54","Column21":"86","Column22":"26","Column23":"","Column24":"","Column25":"26","Column26":"","Column27":"","Column28":"","Column29":"58","Column30":""},
{"Column1":"","Column2":"","Column3":"78","Column4":"35","Column5":"0","Column6":"","Column7":"","Column8":"","Column9":"16","Column10":"86","Column11":"68","Column12":"","Column13":"83","Column14":"","Column15":"24","Column16":"62","Column17":"3","Column18":"","Column19":"84","Column20":"16","Column21":"28","Column22":"34","Column23":"0","Column24":"","Column25":"","Column26":"50","Column27":"61","Column28":"72","Column29":"","Column30":"83"},
{"Column1":"26","Column2":"7","Column3":"10","Column4":"34","Column5":"95","Column6":"","Column7":"64","Column8":"","Column9":"4","Column10":"","Column11":"","Column12":"","Column13":"22","Column14":"4","Column15":"","Column16":"","Column17":"","Column18":"20","Column19":"","Column20":"","Column21":"","Column22":"16","Column23":"12","Column24":"44","Column25":"29","Column26":"","Column27":"","Column28":"35","Column29":"67","Column30":""},
{"Column1":"4","Column2":"25","Column3":"61","Column4":"","Column5":"22","Column6":"","Column7":"","Column8":"","Column9":"","Column10":"55","Column11":"","Column12":"22","Column13":"","Column14":"","Column15":"17","Column16":"","Column17":"","Column18":"","Column19":"","Column20":"40","Column21":"35","Column22":"","Column23":"","Column24":"","Column25":"","Column26":"","Column27":"","Column28":"84","Column29":"9","Column30":""},
{"Column1":"81","Column2":"","Column3":"","Column4":"","Column5":"43","Column6":"51","Column7":"84","Column8":"","Column9":"27","Column10":"32","Column11":"","Column12":"","Column13":"59","Column14":"29","Column15":"4","Column16":"","Column17":"","Column18":"98","Column19":"","Column20":"","Column21":"99","Column22":"53","Column23":"","Column24":"","Column25":"","Column26":"","Column27":"","Column28":"","Column29":"8","Column30":"9"},
{"Column1":"","Column2":"33","Column3":"60","Column4":"11","Column5":"32","Column6":"86","Column7":"29","Column8":"57","Column9":"37","Column10":"5","Column11":"","Column12":"","Column13":"56","Column14":"","Column15":"","Column16":"49","Column17":"60","Column18":"11","Column19":"6","Column20":"67","Column21":"","Column22":"","Column23":"47","Column24":"","Column25":"65","Column26":"","Column27":"77","Column28":"","Column29":"3","Column30":""},
{"Column1":"94","Column2":"60","Column3":"24","Column4":"51","Column5":"52","Column6":"","Column7":"","Column8":"88","Column9":"","Column10":"18","Column11":"","Column12":"","Column13":"55","Column14":"75","Column15":"19","Column16":"","Column17":"","Column18":"70","Column19":"","Column20":"","Column21":"93","Column22":"","Column23":"64","Column24":"","Column25":"","Column26":"17","Column27":"77","Column28":"52","Column29":"87","Column30":""},
{"Column1":"40","Column2":"","Column3":"","Column4":"","Column5":"","Column6":"","Column7":"76","Column8":"","Column9":"96","Column10":"53","Column11":"","Column12":"","Column13":"","Column14":"","Column15":"","Column16":"90","Column17":"9","Column18":"","Column19":"50","Column20":"20","Column21":"","Column22":"75","Column23":"","Column24":"","Column25":"78","Column26":"23","Column27":"67","Column28":"","Column29":"79","Column30":""},
{"Column1":"","Column2":"","Column3":"70","Column4":"","Column5":"","Column6":"","Column7":"4","Column8":"64","Column9":"54","Column10":"","Column11":"","Column12":"12","Column13":"","Column14":"57","Column15":"45","Column16":"44","Column17":"84","Column18":"","Column19":"16","Column20":"","Column21":"","Column22":"12","Column23":"86","Column24":"","Column25":"","Column26":"29","Column27":"43","Column28":"","Column29":"57","Column30":""},
{"Column1":"","Column2":"","Column3":"","Column4":"79","Column5":"","Column6":"75","Column7":"4","Column8":"4","Column9":"","Column10":"49","Column11":"78","Column12":"2","Column13":"","Column14":"","Column15":"","Column16":"29","Column17":"61","Column18":"","Column19":"","Column20":"74","Column21":"","Column22":"","Column23":"73","Column24":"","Column25":"","Column26":"53","Column27":"9","Column28":"","Column29":"","Column30":""},
{"Column1":"35","Column2":"94","Column3":"68","Column4":"29","Column5":"","Column6":"8","Column7":"87","Column8":"","Column9":"44","Column10":"","Column11":"","Column12":"89","Column13":"54","Column14":"","Column15":"7","Column16":"","Column17":"","Column18":"47","Column19":"81","Column20":"88","Column21":"71","Column22":"","Column23":"","Column24":"38","Column25":"26","Column26":"82","Column27":"","Column28":"13","Column29":"35","Column30":"48"},
{"Column1":"","Column2":"58","Column3":"","Column4":"","Column5":"20","Column6":"","Column7":"","Column8":"1","Column9":"","Column10":"47","Column11":"5","Column12":"86","Column13":"28","Column14":"","Column15":"","Column16":"90","Column17":"","Column18":"","Column19":"","Column20":"","Column21":"","Column22":"97","Column23":"31","Column24":"","Column25":"8","Column26":"","Column27":"54","Column28":"","Column29":"17","Column30":"67"},
{"Column1":"2","Column2":"40","Column3":"","Column4":"57","Column5":"","Column6":"59","Column7":"","Column8":"","Column9":"","Column10":"27","Column11":"","Column12":"42","Column13":"","Column14":"53","Column15":"62","Column16":"","Column17":"","Column18":"55","Column19":"85","Column20":"28","Column21":"","Column22":"27","Column23":"44","Column24":"","Column25":"96","Column26":"","Column27":"22","Column28":"58","Column29":"84","Column30":"80"},
{"Column1":"20","Column2":"74","Column3":"19","Column4":"","Column5":"47","Column6":"","Column7":"20","Column8":"51","Column9":"12","Column10":"41","Column11":"","Column12":"98","Column13":"99","Column14":"","Column15":"","Column16":"","Column17":"","Column18":"","Column19":"50","Column20":"55","Column21":"","Column22":"","Column23":"","Column24":"","Column25":"","Column26":"88","Column27":"86","Column28":"","Column29":"60","Column30":"94"},
{"Column1":"","Column2":"44","Column3":"","Column4":"8","Column5":"23","Column6":"","Column7":"","Column8":"","Column9":"45","Column10":"","Column11":"","Column12":"","Column13":"","Column14":"","Column15":"","Column16":"16","Column17":"47","Column18":"86","Column19":"16","Column20":"","Column21":"53","Column22":"55","Column23":"93","Column24":"76","Column25":"95","Column26":"35","Column27":"35","Column28":"","Column29":"","Column30":"57"},
{"Column1":"97","Column2":"","Column3":"5","Column4":"25","Column5":"","Column6":"","Column7":"","Column8":"86","Column9":"90","Column10":"","Column11":"","Column12":"29","Column13":"29","Column14":"","Column15":"","Column16":"20","Column17":"31","Column18":"9","Column19":"20","Column20":"88","Column21":"","Column22":"24","Column23":"81","Column24":"","Column25":"","Column26":"33","Column27":"95","Column28":"7","Column29":"","Column30":""},
{"Column1":"","Column2":"","Column3":"6","Column4":"44","Column5":"","Column6":"66","Column7":"","Column8":"71","Column9":"27","Column10":"","Column11":"82","Column12":"","Column13":"","Column14":"","Column15":"","Column16":"","Column17":"","Column18":"42","Column19":"73","Column20":"","Column21":"72","Column22":"","Column23":"28","Column24":"","Column25":"38","Column26":"32","Column27":"","Column28":"81","Column29":"89","Column30":""},
{"Column1":"1","Column2":"","Column3":"86","Column4":"5","Column5":"48","Column6":"50","Column7":"","Column8":"","Column9":"","Column10":"51","Column11":"25","Column12":"","Column13":"19","Column14":"45","Column15":"","Column16":"","Column17":"","Column18":"93","Column19":"68","Column20":"75","Column21":"","Column22":"39","Column23":"41","Column24":"77","Column25":"20","Column26":"85","Column27":"81","Column28":"33","Column29":"","Column30":""},
{"Column1":"","Column2":"56","Column3":"36","Column4":"50","Column5":"","Column6":"45","Column7":"","Column8":"55","Column9":"","Column10":"","Column11":"56","Column12":"","Column13":"70","Column14":"43","Column15":"","Column16":"","Column17":"","Column18":"","Column19":"","Column20":"22","Column21":"","Column22":"86","Column23":"","Column24":"","Column25":"","Column26":"46","Column27":"41","Column28":"","Column29":"38","Column30":"29"},
{"Column1":"","Column2":"64","Column3":"83","Column4":"46","Column5":"","Column6":"4","Column7":"","Column8":"","Column9":"","Column10":"98","Column11":"","Column12":"","Column13":"","Column14":"71","Column15":"95","Column16":"42","Column17":"33","Column18":"93","Column19":"70","Column20":"","Column21":"49","Column22":"39","Column23":"","Column24":"99","Column25":"11","Column26":"14","Column27":"","Column28":"","Column29":"84","Column30":""},
{"Column1":"25","Column2":"","Column3":"","Column4":"18","Column5":"","Column6":"11","Column7":"","Column8":"85","Column9":"","Column10":"","Column11":"96","Column12":"31","Column13":"","Column14":"18","Column15":"83","Column16":"","Column17":"","Column18":"","Column19":"","Column20":"","Column21":"2","Column22":"49","Column23":"","Column24":"14","Column25":"","Column26":"64","Column27":"99","Column28":"","Column29":"29","Column30":"63"},
{"Column1":"27","Column2":"31","Column3":"","Column4":"80","Column5":"","Column6":"","Column7":"","Column8":"","Column9":"","Column10":"59","Column11":"82","Column12":"27","Column13":"41","Column14":"","Column15":"","Column16":"","Column17":"","Column18":"","Column19":"","Column20":"12","Column21":"98","Column22":"77","Column23":"61","Column24":"","Column25":"","Column26":"18","Column27":"","Column28":"20","Column29":"","Column30":""},
{"Column1":"66","Column2":"69","Column3":"62","Column4":"63","Column5":"","Column6":"","Column7":"","Column8":"","Column9":"42","Column10":"","Column11":"","Column12":"","Column13":"97","Column14":"44","Column15":"","Column16":"60","Column17":"","Column18":"96","Column19":"","Column20":"","Column21":"","Column22":"","Column23":"","Column24":"37","Column25":"60","Column26":"67","Column27":"78","Column28":"","Column29":"7","Column30":""},
{"Column1":"","Column2":"","Column3":"74","Column4":"","Column5":"73","Column6":"","Column7":"","Column8":"","Column9":"","Column10":"59","Column11":"73","Column12":"","Column13":"","Column14":"","Column15":"99","Column16":"","Column17":"","Column18":"","Column19":"68","Column20":"15","Column21":"","Column22":"","Column23":"","Column24":"","Column25":"","Column26":"21","Column27":"","Column28":"","Column29":"73","Column30":""},
{"Column1":"0","Column2":"","Column3":"25","Column4":"10","Column5":"74","Column6":"","Column7":"","Column8":"52","Column9":"","Column10":"31","Column11":"","Column12":"0","Column13":"59","Column14":"62","Column15":"","Column16":"","Column17":"79","Column18":"","Column19":"","Column20":"","Column21":"","Column22":"9","Column23":"","Column24":"85","Column25":"","Column26":"26","Column27":"","Column28":"","Column29":"97","Column30":""},
{"Column1":"","Column2":"65","Column3":"88","Column4":"81","Column5":"16","Column6":"","Column7":"77","Column8":"1","Column9":"57","Column10":"83","Column11":"","Column12":"81","Column13":"","Column14":"45","Column15":"","Column16":"70","Column17":"","Column18":"","Column19":"","Column20":"82","Column21":"39","Column22":"82","Column23":"","Column24":"","Column25":"","Column26":"","Column27":"22","Column28":"","Column29":"16","Column30":""},
{"Column1":"","Column2":"65","Column3":"","Column4":"","Column5":"38","Column6":"71","Column7":"","Column8":"20","Column9":"88","Column10":"","Column11":"","Column12":"50","Column13":"4","Column14":"83","Column15":"18","Column16":"","Column17":"41","Column18":"","Column19":"","Column20":"23","Column21":"24","Column22":"","Column23":"87","Column24":"","Column25":"18","Column26":"","Column27":"22","Column28":"52","Column29":"","Column30":""},
{"Column1":"","Column2":"79","Column3":"","Column4":"","Column5":"","Column6":"","Column7":"","Column8":"","Column9":"","Column10":"82","Column11":"12","Column12":"","Column13":"16","Column14":"11","Column15":"93","Column16":"10","Column17":"","Column18":"","Column19":"92","Column20":"89","Column21":"","Column22":"","Column23":"44","Column24":"","Column25":"","Column26":"41","Column27":"53","Column28":"","Column29":"","Column30":""},
{"Column1":"","Column2":"","Column3":"82","Column4":"","Column5":"56","Column6":"72","Column7":"","Column8":"","Column9":"19","Column10":"","Column11":"33","Column12":"","Column13":"","Column14":"","Column15":"","Column16":"78","Column17":"","Column18":"","Column19":"93","Column20":"","Column21":"","Column22":"","Column23":"","Column24":"","Column25":"","Column26":"79","Column27":"","Column28":"","Column29":"15","Column30":""},
{"Column1":"","Column2":"54","Column3":"4","Column4":"14","Column5":"","Column6":"45","Column7":"94","Column8":"22","Column9":"","Column10":"","Column11":"","Column12":"45","Column13":"","Column14":"","Column15":"","Column16":"1","Column17":"51","Column18":"","Column19":"59","Column20":"51","Column21":"67","Column22":"","Column23":"","Column24":"72","Column25":"8","Column26":"24","Column27":"62","Column28":"","Column29":"","Column30":""}
]
